tests/ui-tests/tests/cases/enum-guess-validation.rs

```diff
@@ -0,0 +1,20 @@
+use super::{query_dsl, types::*};
```

Comment:
Don't think you need these imports - they're adding an extra error to the output.